updating/index.adoc

```diff
@@ -0,0 +1,81 @@
+* xref:../updating/update-using-custom-machine-config-pools.adoc#update-using-custom-machine-config-pools-pause_update-using-custom-machine-config-pools[Pausing the machine config pools]
+* xref:../updating/update-using-custom-machine-config-pools.adoc#update-using-custom-machine-config-pools-update_update-using-custom-machine-config-pools[Performing the cluster update]
+* xref:../updating/update-using-custom-machine-config-pools.adoc#update-using-custom-machine-config-pools-unpause_update-using-custom-machine-config-pools[Unpausing the machine config pools]
+* xref:../updating/update-using-custom-machine-config-pools.adoc#update-using-custom-machine-config-pools-mcp-remove_update-using-custom-machine-config-pools[Moving a node to the original machine config pool]
+
+['updating-clusters-overview-update-cluster-with-rhel-compute-machines']
+== Updating a cluster that includes RHEL compute machines
+xref:../updating/updating-cluster-rhel-compute.adoc#updating-cluster-rhel-compute[Updating a cluster that includes RHEL compute machines]:  You can update an {product-title} cluster. If your cluster contains {op-system-base-full} machines, you must update those machines. You can perform the following actions:
+
+* xref:../updating/updating-cluster-rhel-compute.adoc#updating-cluster-rhel-compute-hooks[Optional: Adding hooks to perform Ansible tasks on RHEL machines]
+* xref:../updating/updating-cluster-rhel-compute.adoc#rhel-compute-updating-minor_updating-cluster-rhel-compute[Updating RHEL compute machines in your cluster]
+
+['updating-clusters-overview-update-restricted-network-cluster']
+== Updating a restricted network cluster
+xref:../updating/updating-restricted-network-cluster.adoc#updating-restricted-network-cluster[Updating a restricted network cluster]: A restricted network environment is one in which your cluster nodes cannot access the internet. You can update a restricted network {product-title} cluster by using the `oc`. For more information, check the following items:
+
+* xref:../updating/updating-restricted-network-cluster.adoc#updating-restricted-network-mirror-host[Preparing your mirror host]
+* xref:../updating/updating-restricted-network-cluster.adoc#installation-adding-registry-pull-secret_updating-restricted-network-cluster[Configuring credentials that allow images to be mirrored]
+* xref:../updating/updating-restricted-network-cluster.adoc#update-mirror-repository_updating-restricted-network-cluster[Mirroring the {product-title} image repository]
+* xref:../updating/updating-restricted-network-cluster.adoc#updating-restricted-network-image-signature-configmap[Creating the image signature config map]
+* xref:../updating/updating-restricted-network-cluster.adoc#machine-health-checks-pausing_updating-restricted-network-cluster[Pausing a MachineHealthCheck resource]
```

Comment:
We don't have this step in v4.7 [doc.](https://docs.openshift.com/container-platform/4.7/updating/updating-restricted-network-cluster.html)

Reply:
Addressed

Reply:
ack